### PR TITLE
[Crystal/en] Crystal language

### PR DESCRIPTION
**Pull request by veelenga**
It would be awesome to be able to learn [Crystal](http://crystal-lang.org/) langauge in Y minutes too :smiley_cat: .

WIP but it's better to get feedback earlier.

### CODE REVIEW

File: crystal.html.markdown [hunk 1]
```diff
@@ -0,0 +1,164 @@
+---
+language: crystal
+filename: learncrystal.rb
```
Comment (vendethiel) [inline crystal.html.markdown]: why `.rb`? the triple-``` ` ``` should be enough

**Comment by onebree** [comment]
I think it is important to note how Crystal is derived from Ruby. A little history is always good for newer languages!

**Comment by veelenga** [comment]
Good point. Gonna add this note and finish soon

File: crystal.html.markdown [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+tmp["foo"] = "bar"
+tmp
+
+# Ranges
+
+1..10                  #=> Range(Int32, Int32)
+Range.new(1..10).class #=> Range(Int32, Int32)
+
+# Can be inclusive or exclusive
+3..5.to_a  #=> [3, 4, 5]
```
Comment (asterite) [inline crystal.html.markdown]: Should be `(3..5).to_a`, otherwise it's parsed as `3 .. 5.to_a` (in Ruby too!)

Comment (asterite) [inline crystal.html.markdown]: And also examples below using range should use parentheses

**Comment by vendethiel** [comment]
Seems like it's not being worked on anymore. I'll close this one for now, I'll reopen it if you work on that again :). Thank you!

**Pull request closed without merging**